memcache/memcache.go

```diff
@@ -380,20 +385,19 @@ func (c *Client) IsACacheEmpty() (bool, error) {
 				return false, err
 			}
 			parsedResponse = strings.Split(string(response), "\n")
-			if len(parsedResponse) < 64 {
+			if len(parsedResponse) != 81 {
```

Comment:
Add in-line comment explaining your explicit choice/use of the number 81; saves time and research for those looking at code later, wondering wtf it means and what it does.